test/known_problems/should_pass/map_union.erl

```diff
@@ -0,0 +1,6 @@
+-module(map_union).
+
+-compile(export_all).
+
+-spec union_value2(#{a => b | c}) -> #{a => b} | #{a => c}.
```

Comment:
This can be solved in the `normalize` function I suppose. Merging maps within a union, like we merge integer intervals, etc. What do you think about that?

Reply:
Solving this in the same way as we handle unions of integers is the right thing to do. I agree.